### PR TITLE
Add autoconsent animation to URL bar

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/488551667048375/1202822739179123/f 

### Description
This PR adds the autoconsent cookies handled animation to the URL bar

### Steps to test this PR

- [x] Install the app and go to `mailchimp.com`, dismiss the onboarding dialog.
- [x] Refresh the page
- [x] You should the see the autoconsent dialog, select manage cookie pop ups.
- [x] After a second or so you should see the cookies animation in the URL bar.
- [x] Go to `cnn.com`
- [x] You should the trackers animation followed by the cookies animation
- [x] Rejecting the cookies will trigger a page refresh
- [x] You should not see any weird glitching animations, during the refresh of the page.
- [x] Go to `http://privacy-test-pages.glitch.me/features/autoconsent`
- [x] You should see the animation
- [x] Refresh the page and tap on the URL bar (to the right of the animation) while the animation is running
- [x] The animation should stop and you should see the URL address selected.
- [x] Refresh the page and you should see the animation again.
- [x] Change to dark mode and refresh the page.
- [x] You should see the animation in dark mode.

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2128** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2128" target="_blank">Graphite</a> 
    * **PR #2129** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2129" target="_blank">Graphite</a> 
    * **PR #2155** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2155" target="_blank">Graphite</a> 
    * **PR #2178** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2178" target="_blank">Graphite</a> 
    * **PR #2181** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2181" target="_blank">Graphite</a> 
    * **PR #2189** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2189" target="_blank">Graphite</a> 
    * **PR #2212** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2212" target="_blank">Graphite</a>  👈
      * **PR #2188** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2188" target="_blank">Graphite</a> 
      * **PR #2203** <a href="https://app.graphite.dev/github/pr/duckduckgo/android/2203" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/android/2212?utm_source=stack-comment).

File: common-ui/src/main/res/drawable/cookies_text_background.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!--
+  ~ Copyright (c) 2018 DuckDuckGo
```
Comment (joshliebe) [inline common-ui/src/main/res/drawable/cookies_text_background.xml:2]: Should be 2022

File: common-ui/src/main/res/drawable/cookies_dummy_background.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ Copyright (c) 2018 DuckDuckGo
```
Comment (joshliebe) [inline common-ui/src/main/res/drawable/cookies_dummy_background.xml:2]: And this one

File: common-ui/src/main/res/drawable/cookies_text_background.xml (hunk 1 above)
Comment (marcosholgado) [reply common-ui/src/main/res/drawable/cookies_text_background.xml:2]: copy&pasta ftw!

**Review by joshliebe — APPROVED** [review]
Looks great and works as expected 👍

**Pull request merged**